extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -310,17 +332,47 @@ export class AmpAutocomplete extends AMP.BaseElement {
     const ampdoc = this.getAmpDoc();
     const policy = UrlReplacementPolicy.ALL;
     const itemsExpr = this.element.getAttribute('items') || 'items';
-    return batchFetchJsonFor(ampdoc, this.element, {
-      expr: itemsExpr,
-      urlReplacement: policy,
-    }).catch(() => {
-      user().warn(
-        TAG,
-        'Expected key "%s" in data but found nothing. Rendering empty results.',
-        itemsExpr
-      );
-      return [];
-    });
+    if (this.isSsr_) {
+      return requestForBatchFetch(
+        this.element,
+        policy,
+        /* refresh */ false
+      ).then(r => {
+        const request = r;
```

Comment:
Why does this need to be aliased as a variable? Why not just call the parameter `request` to begin with?

Reply:
Good catch, fixed!